c/driver_manager/adbc_driver_manager.cc

```diff
@@ -84,6 +84,37 @@ void SetError(struct AdbcError* error, const std::string& message) {
   error->release = ReleaseError;
 }
 
+// Copies src_error into error and releases src_error
+void SetError(struct AdbcError* error, struct AdbcError* src_error) {
+  if (!error) return;
+  if (error->release) error->release(error);
+
+  if (src_error->message) {
+    size_t message_size = strlen(src_error->message);
+    error->message = new char[message_size];
+    std::memcpy(error->message, src_error->message, message_size);
+    error->message[message_size] = '\0';
+  } else {
+    error->message = nullptr;
+  }
+
+  error->release = ReleaseError;
+  if (src_error->release) {
+    src_error->release(src_error);
+  }
+}
+
+struct OwnedError {
+  struct AdbcError error {
+    ADBC_ERROR_INIT
+  };
```

Comment:
huh, the formatter wants this? This threw me for a loop since I would've expected `struct AdbcError error{ADBC_ERROR_INIT}` or `struct AdbcError error = ADBC_ERROR_INIT`. Though ultimately it's not a big deal.

Reply:
ah I'm guessing it's ambiguous to the formatter since without parsing out the types you can't tell if that's a struct declaration or a field declaration

Reply:
...except the field name is right there so ...??? anyways it seems `=` is better to the formatter!